string type
string color
string name
string location
string size
